release-notes/opendistro-for-elasticsearch.security-kibana.release-notes-1.8.0.0.md

```diff
@@ -0,0 +1,9 @@
+## Open Distro for Elasticsearch 1.8.0 Release Notes
```

Comment:
Is there supposed to have a date here similar to other files?

Reply:
After OD 1.7.0 release, we decided to create release notes using standard guidelines. See https://github.com/opendistro-for-elasticsearch/opendistro-build/tree/master/release-notes as an example. Therefore release note for OD 1.8 does not have date. Going forward we will follow the same guidelines.